var/spack/repos/builtin/packages/py-tensorboard/package.py

```diff
@@ -32,22 +37,34 @@ class PyTensorboard(Package):
     depends_on('py-futures@3.1.1:', type=('build', 'run'), when='^python@:2')
     depends_on('py-grpcio@1.24.3:', type=('build', 'run'), when='@2.3:')
     depends_on('py-grpcio@1.23.3:', type=('build', 'run'), when='@2.2')
-    depends_on('py-google-auth@1.6.3:1', type=('build', 'run'))
+    depends_on('py-google-auth@1.6.3:1', type=('build', 'run'), when='@:2.6')
+    depends_on('py-google-auth@1.6.3:2', type=('build', 'run'), when='@2.7:')
     depends_on('py-numpy@1.12.0:', type=('build', 'run'))
     depends_on('py-protobuf@3.6.0:', type=('build', 'run'))
-    depends_on('py-six@1.10.0:', type=('build', 'run'))
+    depends_on('py-six@1.10.0:', type=('build', 'run'), when='@:2.4')
     depends_on('py-werkzeug@0.11.15:', type=('build', 'run'))
-    depends_on('py-wheel', type=('build', 'run'))
-    depends_on('py-wheel@0.26:', type=('build', 'run'), when='@0.6: ^python@3:')
     depends_on('py-google-auth-oauthlib@0.4.1:0.4', type=('build', 'run'))
-    depends_on('py-tensorboard-plugin-wit@1.6.0:', type=('build', 'run'), when='@2.2.0:')
+
+    # py-tensorboard-plugin-wit only builds with versions of bazel older than
+    # what is needed for py-tensorboard-2.5 and greater. Work around that for
+    # now by using a wheel.
+    depends_on('py-tensorboard-plugin-wit@1.6.0:', type=('build', 'run'))
+
+    depends_on('py-tensorboard-data-server@0.6.0:0.6', type=('build', 'run'), when='@2.5:')
```

Comment:
```suggestion
    depends_on('py-tensorboard-data-server@0.6', type=('build', 'run'), when='@2.5:')
```
This should also work?